charts/ark-cloudconfig/values.yaml

```diff
@@ -0,0 +1,74 @@
+# Default values for ark-cloudconfig.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+port: 9999
+image:
+  repository: 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_cloudconfig
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "20210809-d69903c"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+
+podAnnotations: {}
+
+podSecurityContext: {}
+
+livenessProbe:
+  enabled: true
+  path: "/"
+  initialDelaySeconds: 180
+  periodSeconds: 20
+  timeoutSeconds: 5
+  failureThreshold: 6
+  successThreshold: 1
+
+securityContext: 
+  runAsNonRoot: true
+  runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+activemq:
+  externalHost: true
+  brokerUrl: ""
+  brokerPort: ""
+  username: ""
+  password: ""
+commonLabels: {}
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  limits:
+    cpu: 1
+    memory: 953Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+# Requires a Network File System 
+# AWS: EFS 
```

Comment:
What are those comments?

Reply:
Cloudconfig has a NFS dependency. As this chart currently been tested and deployed with EKS added EFS as network file storage. Later when tested with other platforms(minikube, on-prem clusters) will add those as dependencies.

Reply:
All right. I would suggest you make that comment clearer, then.